store/sql_upgrade.go

```diff
@@ -200,6 +200,9 @@ func UpgradeDatabaseToVersion35(sqlStore *SqlStore) {
 		// The rest of the migration from Filenames -> FileIds is done lazily in api.GetFileInfosForPost
 		sqlStore.CreateColumnIfNotExists("Posts", "FileIds", "varchar(150)", "varchar(150)", "[]")
 
+		// Create Team Description column
+		sqlStore.CreateColumnIfNotExists("Teams", "Description", "varchar(255)", "varchar(255)", "")
```

Comment:
Can you rebase this with latest `master` then there should be a `UpgradeDatabaseToVersion36` you can move this to, since 3.5 is already out